tests/queries/0_stateless/03723_parquet_prefetcher_read_big_at.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS t_parquet_prefetcher_read_big_at;
+
+CREATE TABLE t_parquet_prefetcher_read_big_at (a Int32, c String)
+ENGINE = S3(s3_conn, format='Parquet');
+
+INSERT INTO t_parquet_prefetcher_read_big_at
+    SELECT number, toString(number)
+    FROM system.numbers
+    LIMIT 2
+SETTINGS s3_truncate_on_insert=1;
+
+SELECT * FROM t_parquet_prefetcher_read_big_at;
```

Comment:
Need to add `ORDER BY a, c` otherwise test will be flaky.

Reply:
thank you, done